### PR TITLE
chore: update PR template with PON instructions

### PR DESCRIPTION
**Pull request by mathiasbio**
## Description

This PR is created to adress the deviation: https://github.com/Clinical-Genomics/Deviations/issues/666 which was created due to inclusion of tumor samples and wrong bait-set version prepped samples in the creation of the myeloid PoN. 

These are changes to the PR template to highlight the need to verify that the correct samples are included in the PoN. 

#### Added

- New steps to the PR template for PoN sample selection verification

## Documentation

- [x] N/A
- [ ] Updated Balsamic documentation to reflect the changes as needed for this PR.
    - [Document Name]

### Feature Tests

- [x] N/A
- [ ] Test [Description]
    - [Screenshot]

### Pipeline Integrity Tests

- **Report deliver (generation of the `.hk` file)**
    - [x] N/A
    - [ ] Verified
- **TGA T/O Workflow**
    - [x] N/A
    - [ ] Verified
- **TGA T/N Workflow**
    - [x] N/A
    - [ ] Verified
- **UMI T/O Workflow**
    - [x] N/A
    - [ ] Verified
- **UMI T/N Workflow**
    - [x] N/A
    - [ ] Verified
- **WGS T/O Workflow**
    - [x] N/A
    - [ ] Verified
- **WGS T/N Workflow**
    - [x] N/A
    - [ ] Verified
- **QC Workflow**
    - [x] N/A
    - [ ] Verified
- **PON Workflow**
    - [x] N/A
    - [ ] Verified

## Clinical Genomics Stockholm

<details>
<summary></summary>

### Documentation

- **Atlas documentation**
    - [ ] N/A
    - [x] Updated: [Updated atlas PON creation instructions](https://github.com/Clinical-Genomics/atlas/pull/2796)
- **Web portal for Clinical Genomics**
    - [x] N/A
    - [ ] Updated: [Link]

### User Changes

- [x] N/A
- [ ] This PR affects the output files or results.
    - [ ] User feedback is considered unnecessary because [Justification].
    - [ ] Affected users have been included in the development process and given a chance to provide feedback.

### Infrastructure Changes

- **Stored files in Housekeeper**
    - [x] N/A
    - [ ] Updated: [Link]
- **CG (CLI and delivered/uploaded files)**
    - [x] N/A
    - [ ] Updated: [Link]
- **Servers (configuration files on Hasta)**
    - [x] N/A
    - [ ] Updated: [Link]
- **Scout interface**
    - [x] N/A
    - [ ] Updated: [Link]

</details>

## Checklist

> [!IMPORTANT]  
> Ensure that all checkboxes below are ticked before merging.

### For Developers

- **PR Description**
    - [x] Provided a comprehensive description of the PR.
    - [x] Linked relevant user stories or issues to the PR.
- **Documentation**
    - [x] Verified and updated documentation if necessary.
- **Tests**
    - [ ] Described and tested the functionality addressed in the PR.
    - [ ] Ensured integration of the new code with existing workflows.
    - [ ] Confirmed that meaningful unit tests were added for the changes introduced.
    - [ ] Checked that the PR has successfully passed all relevant code smells and coverage checks.
- **Review**
    - [ ] Addressed and resolved all the feedback provided during the code review process.
    - [ ] Obtained final approval from designated reviewers.

### For Reviewers

- **Code**
    - [ ] Code implements the intended features or fixes the reported issue.
    - [ ] Code follows the project's coding standards and style guide.
- **Documentation**
    - [ ] Pipeline changes are well-documented in the CHANGELOG and relevant documentation.
- **Tests**
    - [ ] The author provided a description of their manual testing, including consideration of edge cases and boundary
      conditions where applicable, with satisfactory results.
- **Review**
    - [ ] Confirmed that the developer has addressed all the comments during the code review.

### CODE REVIEW

**Comment by mathiasbio** [comment]
Do you mean the old myeloid PON? Because it's a bit unclear at the moment if it's better to have it or not to have it, I uploaded the same sample both with and without the PON to gens-stage and the copy-number profile did look a bit cleaner with the PoN. So I don't know if it should be removed 🤔 maybe the issue is more that it could be improved

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -91,6 +91,13 @@
     - [x] N/A
     - [ ] Updated: [Link]
 
+### Panel of Normal specific criteria
+
+<!-- If the PR includes a new PoN adhere to the criteria below. -->
+
+- [ ] The PR includes the addition of a new Panel of Normals
+- [ ] The samples have been verified to adhere to the sample selection criteria on Atlas PoN creation instructions for balsamic
```
[the remark this answers is not in the document]
Comment (mathiasbio) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done!

[the remark this answers is not in the document]
Comment (mathiasbio) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done!

**Review by ivadym — APPROVED** [review]

**Comment by ivadym** [comment]
> Do you mean the old myeloid PON? Because it's a bit unclear at the moment if it's better to have it or not to have it, I uploaded the same sample both with and without the PON to gens-stage and the copy-number profile did look a bit cleaner with the PoN. So I don't know if it should be removed 🤔 maybe the issue is more that it could be improved

I see.. If it still brings value, let's keep using the old PON until we have a new version. Good job on documenting the deviation as well 💯

**Pull request merged**